core/js/js.js

```diff
@@ -1153,7 +1155,7 @@ OC.Notification = {
 		if (options.type) {
 			$row.addClass('type-' + options.type);
 		}
-		if (options.type === 'error') {
+		if (options.showCloseButton !== true) {
```

Comment:
```suggestion
		if (options.showCloseButton === true) {
```
Or do I not understand?

Reply:
you are right. done